src/components/ProfileView.jsx

```diff
@@ -314,6 +345,11 @@ const MailConfirmationCode = translate()(({
         <p>{t('ProfileView.twofa.modal.nocode_claude')}<a href='mailto:claude@cozycloud.cc'>claude@cozycloud.cc</a></p>
       </div>
     </div>
+    {
+      fields.email.value === '' && <p className={styles['coz-form-errors']}>
```

Comment:
Does ` fields.email.value` can be `null` ? Does it still supposed to work ?

Reply:
value is set by default to '' by the reducer https://github.com/cozy/cozy-settings/blob/master/src/reducers/fields.js#L12 I'm not really palcid and enthusiastic to change this feature

Reply:
Yep, but if we only consider falsy value, it will continue to work and will also be future-proof. But if you not convinced, let's go with it (It is a concern that should be discussed and statuated by our  - unexisting - style guide).